cupsctl, lpq, lprm, lpadmin, lpoptions: add pages (#11745)
tldr-pages/tldr: pages/common/cupsctl.md

# cupsctl

> > Update or query a server's cupsd.conf.

Reviewer: ```suggestion
> Update or query a server's `cupsd.conf`.
```
We wrap files inside backticks (as suggested in the style guide).

> More information: <https://www.cups.org/doc/man-cupsctl.html>.

> - Display current configuration values:

Reviewer: ```suggestion
- Display the current configuration values:
```

`cupsctl`

- Display the configuration values of a specific server:

`cupsctl -h {{server[:port]}}`

> - Display current configuration values [E]ncrypting the connection to the scheduler:

Reviewer: ```suggestion
- Enable encryption on the connection to the scheduler:
```

`cupsctl -E`

> - Enable or disable debug logging to the error_log file:

Reviewer: ```suggestion
- Enable or disable debug logging to the `error_log` file:
```

> `cupsctl {{--debug-logging|--no-debug-loging}}`

Reviewer: ```suggestion
`cupsctl {{--debug-logging|--no-debug-logging}}`
```

> - Enable or disable debug logging to the error_log file:
> 
> `cupsctl --debug-logging|--no-debug-loging`
> 
> - Enable or disable remote administration:
> 
> `cupsctl --remote-admin|--no-remote-admin`

Reviewer 2: ```suggestion
- Enable or disable debug logging to the error_log file:

`cupsctl {{--debug-logging|--no-debug-loging}}`

- Enable or disable remote administration:

`cupsctl {{--remote-admin|--no-remote-admin}}`
```

To make clear that this is not the shell pipe `|` operator.
